testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/ExtractJsonResponseAction.scala

```diff
@@ -61,17 +65,27 @@ object ExtractJsonResponseAction extends ResponseActions {
 
     val action = fromString(responseAction.name.toLowerCase).getOrElse(None)
     action match {
-      case StringFromList =>
-        val cacheKey = responseAction.params("cacheKey")
-        val listIndex = responseAction.params("listIndex").toInt
+      case StringFromList=>
```

Comment:
```suggestion
      case StringFromList =>
```

Reply:
Suggestion fixed in commit 50143932f2097dbcc40334baae0c69a39fcffa54.